packages/client/components/ActionMeeting.tsx

```diff
@@ -72,6 +72,7 @@ const ActionMeeting = (props: Props) => {
         />
       </ResponsiveDashSidebar>
       <MeetingArea>
+        <Suspense fallback={''}>
         <Phase
           meeting={meeting}
           toggleSidebar={toggleSidebar}
```

Comment:
I noticed eslint/prettier errors about formatting here, it'd be great to fix that before merging 🙏